CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to catapult-explorer
+
+First off, thank you for considering contributing to catapult-explorer.
+
+The catapult-explorer is an open source project and we love to receive contributions from our community — you!
+
+Following these guidelines helps to communicate that you respect the time of
+the developers managing and developing this open source project. In return,
+they should reciprocate that respect in addressing your issue, assessing changes,
+and help you finalize your pull requests.
+
+## Error reports
+
+If you think you have found an error in the catapult-explorer, first make sure that you
+are testing against the latest version of catapult-explorer - your issue may already
+have been fixed. If not, search our issues list on GitHub in case a similar
+issue has already been opened.
+
+It is very helpful if you can prepare a reproduction of the error. It makes it easier to
+find the problem and to fix it.
+
+Please, take into consideration the next template to report your issue:
+
+    **Describe the error**
+    A clear and concise description of what the error is.
+
+    **To Reproduce**
+    Steps to reproduce the behavior:
+    1. Go to '...'
+    2. Click on '....'
+    3. Scroll down to '....'
+    4. See error
+
+    **Expected behavior**
+    A clear and concise description of what you expected to happen.
+
+    **Screenshots**
+    If applicable, add screenshots to help explain your problem.
+
+Open a new issue [here][github-issues].
+
+## Contributing code and documentation changes
+
+To expand and improve the catapult-explorer, please find or open an issue about it first.
+
+Talk about what you would like to do. It may be that somebody is already working on it,
+or that there are particular issues that you should know about before implementing the change.
+
+We enjoy working with contributors to get their pull-requests accepted.
+
+### Contributing License Notice
+
+When you contribute code, you affirm that the contribution is your original work and that you license the work to the project under the project's open source license.
+
+Whether or not you state this explicitly, by submitting any copyrighted material via pull request, email, or other means you agree to license the material under the project's open source license and warrant that you have the legal authority to do so.
+
+### Submitting your changes
+
+Once your changes and tests are ready to submit for review:
+
+1. Test your changes.
```

Comment:
Consider linking or explaining how to test the changes (e.g. how to setup + how to run the test suite).